zkevm-circuits/src/bytecode_circuit/bytecode_unroller.rs

```diff
@@ -716,27 +680,45 @@ mod tests {
         }
 
         fn configure(meta: &mut ConstraintSystem<F>) -> Self::Config {
-            Config::configure(meta, MyCircuit::r())
+            let bytecode_table = BytecodeTable {
+                code_hash: meta.advice_column(),
+                tag: meta.advice_column(),
+                index: meta.advice_column(),
+                is_code: meta.advice_column(),
+                value: meta.advice_column(),
+            };
+
+            let randomness = power_of_randomness_from_instance::<_, 1>(meta);
+            let keccak_table = KeccakTable::construct(meta);
+
+            Config::configure(meta, randomness[0].clone(), bytecode_table, keccak_table)
         }
 
         fn synthesize(
             &self,
             config: Self::Config,
             mut layouter: impl Layouter<F>,
         ) -> Result<(), Error> {
-            config.load(&mut layouter, &self.bytecodes)?;
-            config.assign(layouter, self.size, &self.bytecodes)?;
+            config.load(&mut layouter)?;
+            config.keccak_table.load(
+                &mut layouter,
+                self.bytecodes.iter().map(|b| b.bytes.as_slice()),
+                self.randomness,
+            )?;
+            config.assign(&mut layouter, self.size, &self.bytecodes, self.randomness)?;
             Ok(())
         }
     }
 
-    fn verify<F: Field>(k: u32, bytecodes: Vec<UnrolledBytecode<F>>, success: bool) {
+    fn verify<F: Field>(k: u32, bytecodes: Vec<UnrolledBytecode<F>>, randomness: F, success: bool) {
         let circuit = MyCircuit::<F> {
             bytecodes,
             size: 2usize.pow(k),
+            randomness,
         };
 
-        let prover = MockProver::<F>::run(k, &circuit, vec![]).unwrap();
+        let instance = vec![vec![randomness; (1 << k) - 7 + 1]];
```

Comment:
7 is the degree? (Maybe either add some comment or use cs.degree() will  be better)

Reply:
I've added a variable and a constant that show what those values are in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/622/commits/7955480c55c5be39fc7efb1710ca2dbf58e34ec8